docs/source/architecture/contracts.rst

```diff
@@ -0,0 +1,151 @@
+Ethereum Contracts
+==================
+
+Contract Listing
+----------------
+
+
+* ``NuCypherToken`` ERC20 token contract
+* ``StakingEscrow`` Holds Ursula's stake, stores information about Ursula's activity, and assigns a reward for participating in the NuCypher network. (The ``Issuer`` contract is part of the ``StakingEscrow``\ )
+* ``PolicyManager`` Holds a policy's fee and distributes fee by periods
+* ``Adjudicator`` Manages `the slashing protocol </architecture/slashing>`_
+* ``WorkLock`` Manages token distribution
+* ``Upgradeable`` Base contract for `upgrading </architecture/upgradeable_proxy_contracts>`_
+* ``Dispatcher`` Proxy to other contracts and provides upgrading of the ``StakingEscrow``\ , ``PolicyManager`` and ``Adjudicator`` contracts
+* ``PreallocationEscrow`` Locks tokens for some predetermined time. Tokens will be unlocked after the specified time and all tokens can be used as stake in the ``StakingEscrow`` contract
+
+Deployment Procedure
+--------------------
+
+This section provides a step-by-step guide of how NuCypher contracts are deployed.
+For a guide of how to deploy these contracts automatically, see the `Deployment guide </guides/deployment_guide>`_.
+
+. Deploy ``NuCypherToken`` with all future supply tokens
```

Comment:
This section is not formatted correctly. Did you intend to have bullet points?

Reply:
Hmm - It looks like it it supposed to be a list.   I used m2r to convert this document - so I'm not surprised if there are some misformatted docs - thanks for helping to find them!

Reply:
:heavy_check_mark: